**genomicc has produced 16 papers with an average altmetric score of 619.**

# Whole-genome sequencing reveals host factors underlying critical COVID-19

This research article outlines key findings from the GenOMICC study, using whole-genome sequencing in **7,491 critically ill individuals compared with 48,400 controls to** **discover and replicate 23 independent variants** that significantly predispose patients to critical COVID-19. This included novel 16 variants which were yet to be identified. The multiple variants found in this study tended to be involved in the two mechanisms which have been proposed as the major drivers of severe disease: failure to control viral replication; or an enhanced tendency towards pulmonary inflammation and intravascular coagulation. This article determines the use of the GenOMMIC study in comparing cases of critical illness against population controls in the highly efficient detection of therapeutically relevant mechanisms of disease.

Altmetric score: 1557 [https://doi.org/10.1038/s41586-022-04576-6]


# Genetic mechanisms of severe Covid-19

In collaboration with the [ISARIC GenOMICC study](https://genomicc.org) we discovered multiple genes that underlie critical illness in Covid-19, including several that led directly to potential therapeutic targets. 
In 2244 Covid cases, we compared severely ill patients with matched members of the population from three other studies (UK Biobank, Generation Scotland and 100,000 Genomes). We found genes involved in two molecular processes - antiviral immunity and lung inflammation - were important in determining the development of severe Covid-19.  We replicated our findings in additional studies (Covid-19 HGI and 23andme). The associations with disease are robustly confirmed in these studies.
Although we know the DNA associations are real, we can't always be sure exactly how these variants lead to disease. The most likely genes underlying each of the four new discoveries are IFNAR2, TYK2, OAS1, DPP9.
The action of some genetic variants is similar the the action of drugs - either increasing or decreasing the amount of a particular molecule or signal. We can use this to predict new treatments. This evidence has influenced the inclusion of baricitinib in the RECOVERY trial.

Altmetric score: 2825 [https://doi.org/10.1038/s41586-020-03065-y]


# Two key takeaways from a year of pandemic research

An invited foreword in the journal of Critical Care Medicine by Dr David Maslove and Prof Kenneth Baillie discussing the scientific achievements over the first year of the pandemic discusses the importance of the GenOMICC consortium. The ability of the study, which was in place well before the pandemic, to rapidly adapt to the changing situation is acknowledged as well as the open format of the trail, allowing other institutions and countries to get involved readily.

Altmetric score: 4 [https://doi.org/10.1097/CCM.0000000000005078]


# Neandertal DNA associated with severe illness with COVID-19

Data from the GenOMICC consortium was used to identify a **genetic variation on chromosome 12 which protects against severe disease** when infected with SARS-CoV-2. This inherited genetic factor has been shown to originate in Neandertals and has been shown to be found in humans all over the world. Previously, Neandertal DNA alterations have been shown to increase the risk of severe disease in COVID-19 but this protective variation can help in our understanding of the virus and disease.

Altmetric score: 2352 [https://doi.org/10.1073/pnas.2026309118]


# Identification of a novel protective genetic variant against severe COVID-19

Researchers investigated the human protein transmembrane protease serine type 2 (TMPRSS2) in COVID-19 due to the knowledge that it is required to activate the virus' spike protein and facilitates entry for the virus into target cells. Using the GenOMICC consortium data from critically ill patients, a **common variant in this protein was shown to reduce the likelihood of developing severe COVID-19**. There are already approved drugs to target TMPRSS2, which are currently used for pancreatitis and esophagitis, which could be used in clinical trials for the treatment of COVID-19.

Altmetric score: 73 [https://doi.org/10.1016/j.retram.2022.103333]


# Rare genetic variants influencing COVID-19 outcomes

The GenOMICC cohort was used to aid in the confirmation of results found in other genome-wide association studies into rare variants which were expected to have the largest impact on COVID-19 outcomes. This study aimed to determine these rare variants to provide additional insights into disease susceptibility and pathogenesis, allowing for the determination of future therapeutic targets. The sharing of data during the pandemic has been key for advancements in the understanding and treatment of COVID-19.

Altmetric score: 131 [https://doi.org/10.1371/journal.pgen.1010367]


# Biomarkers of COVID-19 and technologies to combat SARS-CoV-2

The GenOMICC study was acknowledged as an important innovative approach for drug target determination in a review article discussing the ways in which biomarkers and technologies are being utilised in the COVID-19 pandemic.

Altmetric score: 6 [https://doi.org/10.1016/j.abst.2020.08.001]


# The ACE2 coronavirus receptor

It has been discovered that the SARS-CoV-2 virus enters human cells via the angiotensin-converting enzyme 2 (ACE2). GenOMICC consortium data on severe COVID-19 cases was used in this study to show specific variations in ACE2. The researchers also showed that elevated ACE2 levels show a causal relationship with COVID-19 severity, hospitalization, and infection. This suggests that blocking ACE2 pharmacologically may be promising in the treatment of COVID-19 or its comorbidities.

Altmetric score: 49 [https://doi.org/10.1161/CIRCULATIONAHA.121.057888]


# Endomembrane targeting of human OAS1 p46 augments antiviral activity.

Many host RNA sensors are positioned in the cytosol to detect viral RNA during infection. However, most positive-strand RNA viruses replicate within a modified organelle co-opted from intracellular membranes of the endomembrane system, which shields viral products from cellular innate immune sensors. Targeting innate RNA sensors to the endomembrane system may enhance their ability to sense RNA generated by viruses that use these compartments for replication. Here, we reveal that an isoform of oligoadenylate synthetase 1, OAS1 p46, is prenylated and targeted to the endomembrane system. Membrane localization of OAS1 p46 confers enhanced access to viral replication sites and results in increased antiviral activity against a subset of RNA viruses including flaviviruses, picornaviruses, and SARS-CoV-2. Finally, our human genetic analysis shows that the

Altmetric score: 100 [https://doi.org/10.7554/eLife.71047]


# Clonal hematopoiesis is not significantly associated with COVID-19 disease severity.



Altmetric score: 10 [https://doi.org/10.1182/blood.2022015721]


# A prenylated dsRNA sensor protects against severe COVID-19.

Inherited genetic factors can influence the severity of COVID-19, but the molecular explanation underpinning a genetic association is often unclear. Intracellular antiviral defenses can inhibit the replication of viruses and reduce disease severity. To better understand the antiviral defenses relevant to COVID-19, we used interferon-stimulated gene (ISG) expression screening to reveal that 2′-5′-oligoadenylate synthetase 1 (OAS1), through ribonuclease L, potently inhibits severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). We show that a common splice-acceptor single-nucleotide polymorphism (Rs10774671) governs whether patients express prenylated OAS1 isoforms that are membrane-associated and sense-specific regions of SARS-CoV-2 RNAs or if they only express cytosolic, nonprenylated OAS1 that does not efficiently detect SARS-CoV-2. In hospitalized patients, expression of prenylated OAS1 was associated with protection from severe COVID-19, suggesting that this antiviral defense is a major component of a protective antiviral response.

Altmetric score: 1807 [https://doi.org/10.1126/science.abj3624]


# Common, low-frequency, rare, and ultra-rare coding variants contribute to COVID-19 severity.

The combined impact of common and rare exonic variants in COVID-19 host genetics is currently insufficiently understood. Here, common and rare variants from whole-exome sequencing data of about 4000 SARS-CoV-2-positive individuals were used to define an interpretable machine-learning model for predicting COVID-19 severity. First, variants were converted into separate sets of Boolean features, depending on the absence or the presence of variants in each gene. An ensemble of LASSO logistic regression models was used to identify the most informative Boolean features with respect to the genetic bases of severity. The Boolean features selected by these logistic models were combined into an Integrated PolyGenic Score that offers a synthetic and interpretable index for describing the contribution of host genetics in COVID-19 severity, as demonstrated through testing in several independent cohorts. Selected features belong to ultra-rare, rare, low-frequency, and common variants, including those in linkage disequilibrium with known GWAS loci. Noteworthily, around one quarter of the selected genes are sex-specific. Pathway analysis of the selected genes associated with COVID-19 severity reflected the multi-organ nature of the disease. The proposed model might provide useful information for developing diagnostics and therapeutics, while also being able to guide bedside disease management.

Altmetric score: 18 [https://doi.org/10.1007/s00439-021-02397-7]


# Multi-ancestry fine mapping implicates OAS1 splicing in risk of severe COVID-19.

The OAS1/2/3 cluster has been identified as a risk locus for severe COVID-19 among individuals of European ancestry, with a protective haplotype of approximately 75 kilobases (kb) derived from Neanderthals in the chromosomal region 12q24.13. This haplotype contains a splice variant of OAS1, which occurs in people of African ancestry independently of gene flow from Neanderthals. Using trans-ancestry fine-mapping approaches in 20,779 hospitalized cases, we demonstrate that this splice variant is likely to be the SNP responsible for the association at this locus, thus strongly implicating OAS1 as an effector gene influencing COVID-19 severity.

Altmetric score: 725 [https://doi.org/10.1038/s41588-021-00996-8]


# Mendelian randomisation identifies alternative splicing of the FAS death receptor as a mediator of severe COVID-19.

Severe COVID-19 is characterised by immunopathology and epithelial injury. Proteomic studies have identified circulating proteins that are biomarkers of severe COVID-19, but cannot distinguish correlation from causation. To address this, we performed Mendelian randomisation (MR) to identify proteins that mediate severe COVID-19. Using protein quantitative trait loci (pQTL) data from the SCALLOP consortium, involving meta-analysis of up to 26,494 individuals, and COVID-19 genome-wide association data from the Host Genetics Initiative, we performed MR for 157 COVID-19 severity protein biomarkers. We identified significant MR results for five proteins: FAS, TNFRSF10A, CCL2, EPHB4 and LGALS9. Further evaluation of these candidates using sensitivity analyses and colocalization testing provided strong evidence to implicate the apoptosis-associated cytokine receptor FAS as a causal mediator of severe COVID-19. This effect was specific to severe disease. Using RNA-seq data from 4,778 individuals, we demonstrate that the pQTL at the

Altmetric score: 12 [https://doi.org/10.1101/2021.04.01.21254789]


# A catalog of associations between rare coding variants and COVID-19 outcomes.

Severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2) causes coronavirus disease-19 (COVID-19), a respiratory illness that can result in hospitalization or death. We investigated associations between rare genetic variants and seven COVID-19 outcomes in 543,213 individuals, including 8,248 with COVID-19. After accounting for multiple testing, we did not identify any clear associations with rare variants either exome-wide or when specifically focusing on (i) 14 interferon pathway genes in which rare deleterious variants have been reported in severe COVID-19 patients; (ii) 167 genes located in COVID-19 GWAS risk loci; or (iii) 32 additional genes of immunologic relevance and/or therapeutic potential. Our analyses indicate there are no significant associations with rare protein-coding variants with detectable effect sizes at our current sample sizes. Analyses will be updated as additional data become available, with results publicly browsable at https://rgc-covid19.regeneron.com .

Altmetric score: 57 [https://doi.org/10.1101/2020.10.28.20221804]


# Pan-ancestry exome-wide association analyses of COVID-19 outcomes in 586,157 individuals.

Severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2) causes coronavirus disease 2019 (COVID-19), a respiratory illness that can result in hospitalization or death. We used exome sequence data to investigate associations between rare genetic variants and seven COVID-19 outcomes in 586,157 individuals, including 20,952 with COVID-19. After accounting for multiple testing, we did not identify any clear associations with rare variants either exome wide or when specifically focusing on (1) 13 interferon pathway genes in which rare deleterious variants have been reported in individuals with severe COVID-19, (2) 281 genes located in susceptibility loci identified by the COVID-19 Host Genetics Initiative, or (3) 32 additional genes of immunologic relevance and/or therapeutic potential. Our analyses indicate there are no significant associations with rare protein-coding variants with detectable effect sizes at our current sample sizes. Analyses will be updated as additional data become available, and results are publicly available through the Regeneron Genetics Center COVID-19 Results Browser.

Altmetric score: 183 [https://doi.org/10.1016/j.ajhg.2021.05.017]


